provider/bikeshed/data/debug/list/or@2/es-AR.json

```diff
@@ -15,7 +15,7 @@
       "index_1": 3
     },
     "special_case": {
-      "condition": "o|ho|8|(11([\\.  ]?[0-9]{3})*(,[0-9]*)?([^\\.,[0-9]]|$))",
+      "condition": "^[oO]|[hH][oO]|8|(?:11(?:[\\.  ]?[0-9]{3})*(?:,[0-9]*)?(?:[^\\.,[0-9]]|$))",
```

Comment:
Note: The regex changed. Might account for the size diff. Something about case sensitivity?

Reply:
I changed the regex